assets/css/editor-blocks.css

```diff
@@ -0,0 +1,14 @@
+.travel-search .travel-input {
+    line-height: 1.225rem;
+}
+.travel-search .travel-link {
+    color: #8b58e3
+}
+.travel-input-group {
```

Comment:
Could you please add an empty line above this selector? And similarly for the 2 selectors above this?

Otherwise, it'd be fine to remove the empty line above `.travel-search-input.blocks-rich-text`.